### DISCUSSION TITLE
Using "Infill Combination" as well as "Flush into Object's Infill"

### DISCUSSION THREAD

**Discussion by Meier3283** [OrcaSlicer/OrcaSlicer#2788] [Q&A]
I'm wondering if anyone knows, if by using the two settings above, is there any adjustment to the flushing process, since there is roughly half as much infill being laid down?

I ask, because I'm doing one of my first few two color prints and there is some bleeding.  However, I might have shrunk my purge tower down, and that could certainly have an effect.  I'll be doing another print and ensuring the tower is larger.   

But, am I crazy to think those two settings I mentioned, should be linked?  Meaning, if you use "Infill Combination", should it probably purge more "somewhere else"  ( poop chute, or purge tower) ?

**Comment by non-bin** [comment]
[From the bambu wiki](https://wiki.bambulab.com/en/software/bambu-studio/parameter/strength-advance-settings#infill-combination):

> The logic of the infill combination is to take the maximum integer multiple of the wall layer height as the height of the sparse infill within the range of no more than the nozzle diameter.

For example, if you set **Layer Height** to 0.1mm, and **Infill Combination Max layer height** to 0.3, it will print 2 layers of just the walls at 0.1mm, then a layer with 0.1mm walls as well as 0.3mm infill.

So to answer your question: if there's a color change on a layer that's not printing infill, it won't have any infill to flush into and you might get bleeding
